nablapps/nablaforum/views.py

```diff
@@ -83,6 +89,19 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         user = self.request.user
 
+        if not Channel.objects.exists():
+            Channel.objects.create(name="Nablafeed", is_feed=True)
+
+        # Class channel
+        print(
```

Comment:
Remove

Reply:
Should be changed now, in the most recent commit :)